addons/l10n_ec/models/account_journal.py

```diff
@@ -1,18 +1,28 @@
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class AccountJournal(models.Model):
-
     _inherit = "account.journal"
 
-    l10n_ec_entity = fields.Char(string="Emission Entity", size=3, default="001")
-    l10n_ec_emission = fields.Char(string="Emission Point", size=3, default="001")
+    @api.depends('type', 'l10n_latam_use_documents')
```

Comment:
compute method should be after the fields

Reply:
Done, method moved after fields declaration